css/style.css

```diff
@@ -314,10 +313,12 @@ div#tadd select { width: 220px }
 
 @media only screen and (min-width: 1280px) {
   .desktop { display: inline-block; }
+	.dlg-window {max-width: 60vw;}
 }
 
 @media only screen and (max-width: 1279px) {
   .desktop { display: none; }
+	.dlg-window {max-width: 100vw;}
 }
```

Comment:
`min-width: 1280px` is the bigger screen and `max-width: 1279px` is the smaller screen. What is the reason these are inversed? Also there is a problem with line spacing.

Reply:
Okay, what about something like this instead? Over 1000 pixels is too large for 100% of the viewport.
```
.dlg-window {max-width: 60vw;}

@media only screen and (max-width: 729px) {
  .mobile-search { display: none; }
  .dlg-window {max-width: 100vw;}
}
```
If required, you can add anther media query for 80%. Depending on usability, I might actually recommend this.
```
@media only screen and (min-width: 730px) and (max-width: 1279px) {
  .dlg-window {max-width: 80vw;}
}
```

Reply:
Actually, I moved this `max-width` rule to the end of the `style.css` file, where there are media queries for different screen sizes, to keep consistency with the bootstrap break points.